third_party/xla_client/computation_client.cc

```diff
@@ -138,7 +138,6 @@ void PopulateLocalDevices(XrtComputationClient::Options* options) {
 void AddXrtHostDevices(const std::string& worker_name, int task_no,
                        const std::string& server,
                        const DeviceCountDefaults& device_counts,
-                       std::map<std::string, int>* device_ordinals,
```

Comment:
This is not OK. The device ordinals map allows to a consistent device enumeration when called multiple times, and called with different device types.